src/containers/sign-on/store/sagas.js

```diff
@@ -60,6 +62,8 @@ const messages = {
 // Users ID to filter out of the suggested artists to follow list and to follow by default
 /* user id 51: official audius account */
 const defaultFollowUserIds = new Set([51])
+/* user id 12372: official audius account */
+const stagingDefaultFollowUserIds = new Set([12372])
```

Comment:
can we instead define defaultFollowUserIds based on env?

Otherwise, we're polluting the variable space w/ a staging variable on production builds too

let defaultFollowUserIds = new Set([])
if (process.env.ENV === 'production') defaultFollowUserIds = new Set([51])
if (process.env.ENV === 'staging') defaultFollowUserIds = new Set([12372])